### PR TITLE
Support sending notices to multiple rooms

### PR DESCRIPTION
**Pull request by GordonF42**

### CODE REVIEW

**Comment by vabd** [comment]
Will take a look once #2 is merged.

File: config.sample.yaml [hunk 1]
```diff
@@ -31,5 +31,5 @@ notices:
   #   * {{ .URL }}        The SCS's issue/pull request URL.
   # More information on Go patterns can be found at https://godoc.org/text/template
   pattern: "SCS #{{ .Number }} \"{{ .Title }}\" {{ .Message }}: {{ .URL }}"
-  # The ID of the Matrix room to send notices to.
-  room: "!someid:example.com"
+  # The ID of the Matrix rooms to send notices to.
+  Rooms: ["!someid:example.com"]
```
Comment (vabd) [inline config.sample.yaml]:
```suggestion
  rooms: ["!someid:example.com"]
```

Case matters here.

Comment (GordonF42) [reply config.sample.yaml]: my bad, too much aggressive search & replace

**Comment by vabd** [comment]
Since #2 has been merged and brings huge changes to the code base, I'm merging this one manually.

**Pull request merged**